roll20harn.css

```diff
@@ -613,3 +624,205 @@ input[type=number]::-webkit-inner-spin-button {
     text-align: center;
     font-weight: bold;
 }
+
+
+
+
+
+.sheet-rolltemplate-harn-defend table {
+    
+    font-family: "Acme";
+    text-align: center;
+    background-image: url("https://i.imgur.com/Nnc3ddt.gif");
```

Comment:
same thing here - even more important to host it with the repo. that imagr link could change/go away in the future.